FitGoal/FitGoal/ImageFetcher/ExerciseImageFetcherDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ExerciseImageFetcherDelegate.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 4/6/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class ExerciseImageFetcherDelegate: NSObject, ImageFetcherDelegate {
```

Comment:
Use name that reflects the purpose of the object instead of the protocols it implements.